core/total-conversion-build.js

```diff
@@ -49,11 +46,12 @@ if (!window.PLAYER || !PLAYER.nickname) {
 // possible without requiring scripts.
 document.head.innerHTML = ''
   + '<title>Ingress Intel Map</title>'
-  + '<style>'+'@include_string:style.css@'+'</style>'
-  + '<style>'+'@include_css:external/leaflet.css@'+'</style>'
-//note: smartphone.css injection moved into code/smartphone.js
+  + '<style>' + IITCTool.importString('style.css') + '</style>'
+  // note: smartphone.css injection moved into code/smartphone.js
   + '<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,400,400italic,500,500italic,700,700italic&subset=latin,cyrillic-ext,greek-ext,greek,vietnamese,latin-ext,cyrillic"/>';
 
+IITCTool.importCSS('external/leaflet.css');
```

Comment:
any reason to differenciate the main style and leaflet.css ?

Reply:
leaflet needs "url" parsing.

Reply:
ok
`style.css` might also use `url` at some point tho
also the name matches this use case, importing css.